### DISCUSSION TITLE
Replace specified tag

### DISCUSSION THREAD

**Discussion by D4rkT1m0** [firefly-iii/firefly-iii#8197] [Firefly III]
### Support guidelines

- [X] I've read the [support guidelines](https://github.com/firefly-iii/firefly-iii/blob/main/.github/support.md)
- [X] My request is not listed as [a very good idea, but unfortunately...](https://docs.firefly-iii.org/firefly-iii/more-information/what-its-not/)
- [X] I've used [the search](https://github.com/firefly-iii/firefly-iii/issues?q=is%3Aissue) and this has not been requested before.

### Description

in the tag section, I would like the ability to massively replace a specified tag with another, when some operation have multiple tags. and not only "replace all tag" by this one.

### Solution

this way, operation with multiple tag keep their existing tag when you only want to replace one of them

**Comment by JC5** [comment]
You can make a rule to do that.
